### PR TITLE
[Feature]: Theme toggle component

### PR DESCRIPTION
**Pull request by tux255**

### CODE REVIEW

File: src/components/ThemeToggleSwitch.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    handleThemeChange(!isOn)
+  }
+
+  function handleThemeChange(isOn) {
+    if (isOn) {
+      document.documentElement.classList.remove('dark')
+    } else {
+      document.documentElement.classList.add('dark')
+    }
+  }
```
[the remark this answers is not in the document]
Comment (tux255) [reply src/components/ThemeToggleSwitch.jsx:17]: done

**Review by Nedilko — APPROVED** [review]

**Comment by nedref** [comment]
closes #10

**Comment by Nedilko** [comment]
closes #10

**Comment by Nedilko** [comment]
closes #12

**Pull request merged**